src/pages/Playground/PlaygroundContainer.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useState } from "react";
```

Comment:
so everything from CodeEditor just moved here?

Reply:
The code from CodeEditor that just involved page layout (sidebar, header) was moved to here, and content that's inside main page area was compartmentalized to CodeEditorPage for problems and LearnPage for lessons